views/index.ejs

```diff
@@ -72,6 +72,21 @@
           </div>
         </div>
       </nav>
-    <h1>Create Your VERY OWN Todolist!</h1>
+
+<div class="flex flex-col items-center justify-center h-[calc(100%-4rem)] py-2 overflow-hidden">
```

Comment:
put the calc there to hopefully account for the height difference with the navbar. That way the height of the whole page is the whole page and nothing more (no scrolling)